tests/unsafe-traits.rs

```diff
@@ -0,0 +1,24 @@
+// This is a compile-test to ensure the proc macro can handle unsafe traits.
```

Comment:
We have compile tests inside of `macros/tests/`. I think it's best if these tests went their.

Reply:
Done. Had to add `winapi` dev-dependency to the macros crate though.